### ISSUE TITLE
Performance Issue related to batch size

### ISSUE THREAD

**Issue by LeeHoang2710** [feifeiobama/PatchKD#6]
I’ve been working on reproducing the results and have found the problems.

To train the model faster, I modified the batch size in the train dataloader from 128 to 64. With this adjustment, my results on the first dataset seem comparable to those reported in your paper. However, when testing on subsequent datasets, I observed a significant performance of mAP drop on the first dataset, which fell to 55.7 when I reached the third dataset.
![image](https://github.com/user-attachments/assets/b3a44f31-e9a4-45d2-b850-bd9a3c969c5b)

I am wondering:
1. Does reducing the batch size significantly impact the results across datasets in your experience?
2. Are there any specific adjustments or training strategies you recommend when working with smaller batch sizes to maintain consistent performance in continual learning?
Here is my result:
![image](https://github.com/user-attachments/assets/b3e61d04-1f51-4f74-98a3-3116360683c5)
![image](https://github.com/user-attachments/assets/0a71deff-d9bc-48c7-811d-cd44d2af9252)
Thank you for your support!

**Comment by feifeiobama** [comment]
1. Yes, one of our module (patch relation distillation) is computed within each batch and may be sensitive to batch size. This module is only applied in subsequent tasks, so it will only affect later results, which is consistent with your finding.
2. A possible solution is to adjust the loss weight for patch relation distillation, but I have not experimented with this for smaller batch sizes.
